index.js

```diff
@@ -102,6 +105,8 @@ module.exports = fp(function from (fastify, opts, next) {
         if (!this.sent) {
           if (err.code === 'ERR_HTTP2_STREAM_CANCEL') {
             this.code(503).send(new Error('Service Unavailable'))
+          } else if (err.code === ERR_TIMED_OUT_CODE) {
+            this.code(504).send('Gateway Timeout')
```

Comment:
This should be an error instance.

Reply:
I am currently throwing bunch of different error instances with different error messages in `lib/request.js`. I initially planned to throw the errors to the library users, but with the current approach you suggestion makes a lot of sense 👍.

Reply:
See line 107. Why is line 109 only returning a string?